src/datasources/system/SystemDataSource.ts

```diff
@@ -28,8 +28,8 @@ export class SystemDataSource extends DataSourceBase<SystemQuery> {
       return {
         refId: query.refId,
         fields: [
-          { name: 'Connected', values: [summary.connectedCount], type: FieldType.number },
-          { name: 'Disconnected', values: [summary.disconnectedCount], type: FieldType.number },
+          { name: 'Connected', values: [summary.connectedCount] },
+          { name: 'Disconnected', values: [summary.disconnectedCount] },
```

Comment:
Grafana infers the types correctly, so it's just extra stuff to match against in tests